web/src/components/pages/profile/avatar-setup/avatar-setup.css

```diff
@@ -76,6 +113,23 @@
         padding-left: 22px;
     }
 
+    .recreate-voice {
+        width: 470px;
+        height: 58px;
+        font-family: OpenSans;
+        font-size: 16px;
```

Comment:
It looks like these values are more or less verbatim from Zeplin. I'd recommend not doing that, as many of these are just the default values and create noise in the code. And for some others we also have variable we could use here. For sizes it's often preferable to keep things dynamic, so that they respond to different screen sizes.

Reply:
Ah, I was thinking I have to keep little sync in code with Zeplin. I understand !

Reply:
I see, sorry for the confusion and thanks for the fix!